app/src/main/java/com/ivy/wallet/ui/statistic/level2/ItemStatisticScreen.kt

```diff
@@ -36,6 +36,7 @@ import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.lifecycle.viewmodel.compose.viewModel
+import com.google.firebase.crashlytics.internal.model.ImmutableList
```

Comment:
⚠️ That's some Firebase Crashlytics `ImmutableList` - we need the immutable list from this KotlinX library https://github.com/Kotlin/kotlinx.collections.immutable! Otherwise, the Compose optimization won't work.